src/ontology/edits/oeo-physical.omn

```diff
@@ -6071,34 +6072,27 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/911",
     EquivalentTo: 
         (<http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00010138)
          and (<http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00010140)
-    
-    SubClassOf: 
-        <http://purl.obolibrary.org/obo/BFO_0000015>,
-        OEO_00000532 some OEO_00000006
```

Comment:
The PR deletes the SubClassOf axioms of the class `carbon capture and storage` (OEO_00010141)

Reply:
Thanks, well spotted. I reverted the deletion.